flutter-for-react-native.md

```diff
@@ -0,0 +1,2007 @@
+
+Try it out in [DartPad](https://dartpad.dartlang.org/e708f2cc5504f405a374537242a579b5).
+
+**Note:** Check [here](https://www.dartlang.org/tutorials/language/streams) for more details on Streams. 
+You can refer [here](https://www.dartlang.org/resources/synonyms) for more differences.
+
+## Learn The Basics of Flutter
+### How do I create a Flutter app?
+
+In React Native, you would start off your development process by creating the project using the below command line tool.
+
+<!-- skip -->
+{% prettify shell %}
+$ create-react-native-app {projectname}
+{% endprettify %}
+
+In [Flutter](https://flutter.io/getting-started/), you can use this command line tool to do the same.
+
+<!-- skip -->
+{% prettify shell %}
+$ flutter create {projectname}
+{% endprettify %}
+
+### How do I run my app?
+
+In React Native, you would go to the project directory and use 
+`npm run ios/android` or `yarn run ios/android`.
+In [Flutter](https://flutter.io/getting-started/), if you are using the terminal, then you use the `flutter run` command in the project root directory to run your app on a connected device or simulator. 
+If you are using an IDE like IntelliJ, Android Studio, or VS Code with the Flutter plugin installed then you can use the in-built tools to run the app. 
+
+Refer [here](https://flutter.io/getting-started/) for more details.
+
+### How do I use import statements?
+
+<!-- skip -->
+{% prettify javascript %}
+//React Native
+import React from "react";
+import { StyleSheet, Text, View } from "react-native";
+{% endprettify %}
+
+Contrary to React Native’s way of importing each component as they are used, in Flutter, you import the `material.dart` from the flutter package, which allows you to use any material design widget without explicitly importing it.
+
+<!-- skip -->
+{% prettify dart %}
+import 'package:flutter/material.dart';
+{% endprettify %}
+
+### What is the equivalent of React Native `Hello World` app in Flutter?
+
+Let’s take a look at how we can define the app in their respective frameworks:
+
+<!-- skip -->
+{% prettify javascript %}
+// React Native
+import React from "react";
+import { StyleSheet, Text, View } from "react-native";
+
+export default class HelloWorldApp extends React.Component {
+  render() {
+    return (
+      <View>
+        <Text>Hello World</Text>
+      </View>
+    );
+  }
+}
+{% endprettify %}
+
+<!-- skip -->
+{% prettify dart %}
+// Flutter
+import 'package:flutter/material.dart';
+
+void main() => runApp(new HelloWorldApp());
+
+class HelloWorldApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return new MaterialApp(
+      home: new Scaffold(
+        body: new Column(
+         mainAxisAlignment: MainAxisAlignment.center,
+          children: <Widget>[
+            new Center(
+              child: new Text(
+                'Hello World',
+              ),
+            ),
+          ],
+        ),
+      )
+    );
+  }
+}
+{% endprettify %}
```

Comment:
these two codes snippets aren't equivalent at all. We can dramatically simplify the Flutter version to make it more equivalent to the RN version.